### ISSUE TITLE
Untested with Studio version

### ISSUE THREAD

**Issue by zelikos** [zelikos/davincibox#26]
I've only used the free version of DaVinci Resolve, as that has met my needs enough for my use cases, so using davincibox with the Studio version is another thing I'm unable to test.

Help would of course be appreciated here, but I have to urge caution in doing so. IIUC, the Studio version requires activation with a license key?

1) I don't know if running Resolve Studio in a container would interfere with that
2) Care would have to be taken to **not** delete the container without first deactivating Resolve Studio; I assume recovering from that would be doable through Blackmagic support, but it's still a headache that would be best to avoid

**Comment by rothgar** [comment]
I have davinci installed locally in /opt and also tried with davincibox and I was able to register my studio license.

![image](https://github.com/zelikos/davincibox/assets/371796/7e9fbe93-0306-4023-8750-9b596671329f)

It prompted me to put in my key again but so far it all appears to work fine.

**Comment by not-a-dev-stein** [comment]
18.6 Studio is working for me but one issue I'm facing (which I don't even know where to start looking into to solve) is that the program doesn't seem to be handling audio sources well, where I'm unable to play videos on YouTube or other videos while Resolve is open, and that it doesn't recognize my bluetooth earbuds and keeps playing sound through my laptop speakers.

Likely unrelated to this issue, but sharing the experience anyway.

**Comment by zelikos** [comment]
> EDIT: Installing the `pipewire-alsa` package in the davincibox toolbox fixes it, might be something you'd like to look into doing by default.

`pipewire-alsa` is already included by default as of #29, two months ago.

In a fresh davincibox container:

```
akzel@davincibox:~$ sudo dnf install pipewire-alsa
Copr repo for distrobox-utils owned by kylegospo                                                                                                                                                                                                                                               10 kB/s | 1.5 kB     00:00    
Fedora 39 - x86_64 - Updates                                                                                                                                                                                                                                                                  107 kB/s |  23 kB     00:00    
Fedora 39 - x86_64 - Updates                                                                                                                                                                                                                                                                  3.1 MB/s | 8.3 MB     00:02    
RPM Fusion for Fedora 39 - Free - Updates                                    7.7 kB/s | 2.9 kB     00:00    
RPM Fusion for Fedora 39 - Free - Updates                                    184 kB/s | 142 kB     00:00    
RPM Fusion for Fedora 39 - Nonfree - Updates                                  23 kB/s | 5.8 kB     00:00    
RPM Fusion for Fedora 39 - Nonfree - Updates                                 142 kB/s |  67 kB     00:00    
Package pipewire-alsa-1.0.0-1.fc39.x86_64 is already installed.
Dependencies resolved.
Nothing to do.
Complete!

```

**Comment by not-a-dev-stein** [comment]
Oh shoot, it might have something to do with it, as I had tried davincibox a couple of months back but it didn't work for me at the time.

**Comment by zelikos** [comment]
> Oh shoot, it might have something to do with it, as I had tried davincibox a couple of months back but it didn't work for me at the time.

Yeah, that would be it. Distrobox doesn't automatically pull new versions of a container image unless explicitly configured to, so if you already had the image, any new containers would be using that same old version.

You can run `podman image pull ghcr.io/zelikos/davincibox:latest`, then run `setup.sh` again to have the latest version

**Comment by queequeg0925** [comment]
> I've only used the free version of DaVinci Resolve, as that has met my needs enough for my use cases, so using davincibox with the Studio version is another thing I'm unable to test.
> 
> Help would of course be appreciated here, but I have to urge caution in doing so. IIUC, the Studio version requires activation with an activation key?
> 
>     1. I don't know if running Resolve Studio in a container would interfere with that
> 
>     2. Care should be taken to **not** delete the container without first deactivating Resolve Studio, if possible; I don't know how trying to activate a different installation without first deactivating the previous is actually handled by Resolve Studio, but it's a potential headache that would be best to avoid
> 
> 
> tl;dr: Help wanted on testing Resolve Studio, but be careful

I'm going to give resolve studio a try on my arch system. Activation key should not be an issue. Resolve allows two activation. When you activate a third time, it clears the previous activation automatically.

**Comment by geewiz** [comment]
I've been running Resolve Studio in Distrobox for about two years. Since that Distrobox is so old that it started acting up recently with a known `sudo` issue, I decided to give davincibox a shot. It works nicely!

I'm grateful for your work, @zelikos! :heart:

**Comment by mtwb47** [comment]
Studio also works very well for me. The only issue I've found is that the text scaling is way off, but IDK that that's a Studio issue or not.

**Comment by marbub** [comment]
I am running an older version of DaVinci Box with Studio. The only thing I noticed is that I had to remove libgobject, libmodule, etc. because there were linking errors. I think the origin is that DaVinci Box is based on Fedora instead of Rocky Linux, so the symbols do not match.

**Comment by zelikos** [comment]
> I am running an older version of DaVinci Box with Studio. The only thing I noticed is that I had to remove libgobject, libmodule, etc. because there were linking errors. I think the origin is that DaVinci Box is based on Fedora instead of Rocky Linux, so the symbols do not match.

davincibox builds since **June** have had a fix for the issue with those packages

**Comment by Zindswini** [comment]
I would also like to chime in and say I have also had no issues using studio on Nvidia. All codecs seem to be working, no issues with activation, and no issues with fonts either. Thanks for the great work!

**Comment by fenixstarlord** [comment]
Hello, I'm using Davincibox with studio on Bazzite. 

One issue I've run into, it's unable to recognize USB dongles. The serial key works fine.

**Comment by fenixstarlord** [comment]
Also it appears that audio transcription does not work or is glacially slow. I'm using AMD, so that might be part of the issue.

**Comment by Davphla** [comment]
Worked for me, everything work flawlessly. GPU detected.
Version: DaVinci Resolve Studio 20.0.1
GPU (Discrete): AMD Radeon 780M Graphics

**Comment by sharms** [comment]
On Nvidia I noticed I cannot export most formats (Bluefin / Fedora 43)

**Comment by vignesh-seven** [comment]
```
./resolve: error while loading shared libraries: libgpudetect.so: cannot open shared object file: No such file or directory
```

I am getting this error after trying to start the Studio version 20.3.1  
The free version is working fine tho, GPU detected and everything. (I followed [this to install it and the Studio version](https://techhut.tv/install-davinci-resolve-linux-ubuntu-arch-fedora-davincibox))  

EndeavorOS  
RTX 4060  
Laptop  

I have tried installing [nvidia-container-toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html#with-dnf-rhel-centos-fedora-amazon-linux) inside davincibox but it didn't help.  

Help appreciated :)

**Comment by zelikos** [comment]
> I have tried installing [nvidia-container-toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html#with-dnf-rhel-centos-fedora-amazon-linux) inside davincibox but it didn't help.

`nvidia-container-toolkit` needs to be installed on the host, not in the container.

**Comment by CarlosGT7274** [comment]
ye i have no audio on studio 21.0 someone knows something?

**Comment by Jack-the-Pro101** [comment]
I can confirm it works flawlessly on DaVinci Resolve Studio 21.0.4, all features work, and license activation is without issue.

OS: Aurora (Universal Blue distro)
GPU: NVIDIA Geforce RTX 3070 Ti Laptop GPU (GPU detected)

Note, I had to run the following inside the container in order to install the Reactor package manager using the Lua script
```
sudo dnf copr enable nucleo/openssl1.1 -y
sudo dnf install openssl1.1 -y
```
